### DISCUSSION TITLE
Can't access Scrypted through NAS address

### DISCUSSION THREAD

**Discussion by rodrodesign** [koush/scrypted#200] [Q&A] [answered]
Hello, I need a little help please 

I've done all the installation process through Synology Docker and the container is running, but when I put the https://{IP of nas}:10443/ address, it shows as an invalid address

What can I do to access my Scrypted control panel or log in panel?

Thanks

EDIT: Did a fresh installation and it worked.

**Comment by koush** [comment, marked as answer]
seems to be user setup issue
